src/Core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
```

Comment:
I think this bug is serious enougth to tag a `1.2.1` release as soon as merged. 
WDYT

Reply:
Yes!

Reply:
that were the wrong CHANGELOG
Updated SES's CHANGELOG